src/components/ButtonSection.js

```diff
@@ -125,12 +151,14 @@ export { ButtonSection as PureButtonSection };
 
 const mapStateToProps = state => ({
   smData: state.smData,
-  peaksInstance: state.peaksInstance
+  peaksInstance: state.peaksInstance,
+  showForms: state.showForms
```

Comment:
I think at some point we refactor the naming convention on this Redux reducer `show-forms`.  It's become a catch-all and last few times I see it referenced it's confusing to what it actually does.  

Maybe the reducer and action is renamed to something like `forms`, and `disabled` gets more specific to what is disabled?

Reply:
Yeah, sounds like a good idea. It's kind of vague now, and someone new will be confused. I'll refactor these in the ticket I'm working now.